go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/vishvananda/netlink v1.1.1-0.20201029203352-d40f9887b852
 	golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c
 	google.golang.org/grpc v1.39.0
-	google.golang.org/protobuf v1.27.1
+	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.1.0 // indirect
+	google.golang.org/protobuf v1.26.0
```

Comment:
I assume this change is necessary?

Reply:
it showed up when i built after sync to latest. but it is not necessary. i'll remove it

Reply:
I reverted this change in my code. But this happens when you run bootstrap. Can you please check if the bootstrap script needs to be updated?